lte/gateway/configs/health.yml

```diff
@@ -22,6 +22,6 @@ state_recovery:
 
   # Number of restarts of services_check that triggers recovery process
   restart_threshold: 2
-  interval_check_mins: 3
+  interval_check_mins: 1
```

Comment:
Why this change? The PR description lists systemd issues.

Reply:
Can be removed no idea how that slipped in here